sdk/common/env.go

```diff
@@ -0,0 +1,59 @@
+package common
```

Comment:
Same here, we dont need this in the SDK IMO.

Reply:
@nickolaev ok, thanks. Where I can store the code that will be needed for several sidecars?
Is `side-cars` good folder for it?

Reply:
finally moved to `utils`

Reply:
I still think the contents of this file does not belong to the `sdk`

Reply:
@nickolaev yep, fixed